client/app/components/pages/GeneHome.vue

```diff
@@ -2,252 +2,6 @@
  * GeneHome.vue
  *
  */
-<style lang="sass">
```

Comment:
Would you mind backing out the change that moves styling to the bottom? Then I will be able to see the modified code to introduce the PedigreeGenotypeViz dialog.

Reply:
The only thing I changed here on gene home was moving the styling and taking out that extra v-dialog. I had thought about putting the popup as a home component but ended up deciding to put it in the variantInspectCard. I moved them back to the top but not sure it really helped.